Assets/MixedRealityToolkit.Services/SceneSystem/MixedRealitySceneSystem.cs

```diff
@@ -37,14 +37,28 @@ private enum SceneType
             Lighting = 2,
         }
 
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        /// <param name="registrar">The <see cref="IMixedRealityServiceRegistrar"/> instance that loaded the service.</param>
+        /// <param name="profile">The configuration profile for the service.</param>
+        [Obsolete("This constructor is obsolete (registrar parameter is no longer required) and will be removed in a future version of the Microsoft Mixed Reality Toolkit.")]
         public MixedRealitySceneSystem(
             IMixedRealityServiceRegistrar registrar,
-            MixedRealitySceneSystemProfile profile) : base(registrar, profile)
+            MixedRealitySceneSystemProfile profile) : this(profile)
         {
-            this.profile = profile;
+            Registrar = registrar;
         }
 
-        private MixedRealitySceneSystemProfile profile;
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        /// <param name="profile">The configuration profile for the service.</param>
+        public MixedRealitySceneSystem(
+            MixedRealitySceneSystemProfile profile) : base(profile)
+        { }
+
+        private MixedRealitySceneSystemProfile profile => ConfigurationProfile as MixedRealitySceneSystemProfile;
```

Comment:
Should we cache this cast in the constructor?

Reply:
the base constructor sets the value of ConfigurationProfile

Reply:
should the constructor also set the value of `profile`, so we aren't casting every time this field is used?